### PR TITLE
vso: add page

### LINKED ISSUE #9726: page request: vso
This issue is being tracked at <https://github.com/orgs/Vanilla-OS/projects/2>.

VSO is a utility which allows you to perform maintenance tasks on your Vanilla OS installation.

Link: <https://github.com/Vanilla-OS/vanilla-system-operator>

I will create a PR for it soon.

### PR DESCRIPTION
**Pull request by kbdharun**
Signed-off-by: K.B.Dharun Krishna <kbdharunkrishna@gmail.com>

- [x] The page(s) are in the correct platform directories: `linux`.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known): `1.1.4`**

Closes #9726

All options on this page are listed at https://documentation.vanillaos.org/docs/vso/manpage (I wrote the manpage myself 😄 )

### CODE REVIEW

File: pages/linux/vso.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+> Utility to perform maintenance tasks on Vanilla OS.
+> More information: <https://github.com/Vanilla-OS/vanilla-system-operator>.
+
+- Trigger a system update immediately:
+
+`sudo vso trigger-update --now`
+
+- Create an automated task upon an application's launch:
+
+`vso create-task --name "{{string}}" --description {{string}} --{{command|on-condition-command}} "{{cmd}}" --on-process {{process}}`
```
Comment (EmilySeville7cfg) [inline pages/linux/vso.md:12]:
```suggestion
- Create an automated task upon an application's launch:

`vso create-task --name "{{string}}" --description "{{string}}" --{{command|on-condition-command}} "{{command}}" --on-process {{process}}`
```

Optional fix: unexpected `{{` term, quoted placeholder expected.
Optional fix: unexpected `cmd` contraction, `command` expected.
Optional fix: unexpected `process` term, `integer` expected.

File: pages/linux/vso.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+- Create an automated task to execute during boot:
+
+`vso create-task --name "{{string}}" --description {{string}} --command "{{cmd}}" --on-boot`
```
Comment (EmilySeville7cfg) [inline pages/linux/vso.md:16]:
```suggestion
- Create an automated task to execute during boot:

`vso create-task --name "{{string}}" --description "{{string}}" --command "{{command}}" --on-boot`
```

Almost the same issue as above.

File: pages/linux/vso.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+- Create an automated task to execute during specific battery states:
+
+`vso create-task --name "{{string}}" --description {{string}} --command "{{cmd}}" --{{on-low-battery|on-charge|on-battery|on-full-battery}}`
```
Comment (EmilySeville7cfg) [inline pages/linux/vso.md:20]:
```suggestion
- Create an automated task to execute during specific battery states:

`vso create-task --name "{{string}}" --description "{{string}}" --command "{{command}}" --{{on-low-battery|on-charge|on-battery|on-full-battery}}`
```

The same issue as above.

File: pages/linux/vso.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+`vso create-task --name "{{string}}" --description {{string}} --command "{{cmd}}" --{{at|every}} "{{time_interval}}"`
+
+- Create an automated task which asks for a confirmation before execution:
+
+`vso create-task --name "{{string}}" --description {{string}} --command "{{cmd}}" --need-confirm`
+
+- Create an automated task to execute during network connection or disconnection:
+
+`vso create-task --name "{{string}}" --description {{string}} --command "{{cmd}}" --{{on-network|on-disconnect}}`
```
Comment (EmilySeville7cfg) [inline pages/linux/vso.md:32]:
```suggestion
- Create an automated task to execute during network connection or disconnection:

`vso create-task --name "{{string}}" --description "{{string}}" --command "{{command}}" --{{on-network|on-disconnect}}`
```

The same issue as above.

**Review by EmilySeville7cfg — APPROVED** [review]
No mandatory fixes found, approved.
Thanks for a contribution! 🚀

File: pages/linux/vso.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+`vso create-task --name "{{string}}" --description {{string}} --command "{{cmd}}" --on-boot`
+
+- Create an automated task to execute during specific battery states:
+
+`vso create-task --name "{{string}}" --description {{string}} --command "{{cmd}}" --{{on-low-battery|on-charge|on-battery|on-full-battery}}`
+
+- Create an automated task to execute [at] a specific time interval (hh: mm) or re-execute it [every] time after specific time intervals (m,h,d):
+
+`vso create-task --name "{{string}}" --description {{string}} --command "{{cmd}}" --{{at|every}} "{{time_interval}}"`
```
[the remark this answers is not in the document]
Comment (kbdharun) [reply pages/linux/vso.md]: Done applied all suggestions. I am not sure if "" is required for `{{time_interval}}` or not. I will confirm and get back here.

**Review by EmilySeville7cfg — APPROVED** [review]

**Review by blueskyson — APPROVED** [review]

File: pages/linux/vso.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+
+`vso create-task --name "{{string}}" --description "{{string}}" --command "{{command}}" --on-boot`
+
+- Create an automated task to execute during specific battery states:
+
+`vso create-task --name "{{string}}" --description "{{string}}" --command "{{command}}" --{{on-low-battery|on-charge|on-battery|on-full-battery}}`
+
+- Create an automated task to execute at a specific time interval (hh: mm) or re-execute it every time after specific time intervals (m,h,d):
+
+`vso create-task --name "{{string}}" --description "{{string}}" --command "{{command}}" --{{at|every}} {{time_interval}}`
```
Comment (navarroaxel) [inline pages/linux/vso.md:24]: I think this example should be split into 2. is the time in 24hs format or am/pm?

Comment (kbdharun) [reply pages/linux/vso.md:24]: Yeah, but the page already has 8 examples, maybe I will remove this one and change it to a different one for update-check.

It's in 24 hours format afaik.

Comment (kbdharun) [reply pages/linux/vso.md]: Done, updated it.

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**